README.md

```diff
@@ -45,39 +45,26 @@ High velocity native mobile development requires us to adopt continuous integrat
 
 The most difficult part of automated testing on mobile is the tip of the testing pyramid - E2E. The core problem with E2E tests is flakiness - tests are usually not deterministic. We believe the only way to tackle flakiness head on is by moving from black box testing to gray box testing. That’s where Detox comes into play.
 
-- **Cross Platform:** Write cross-platform end-to-end tests in JavaScript. Currently supports iOS and Android.
+- **Cross Platform:** Write end-to-end tests in JavaScript for React Native apps (Android & iOS).
 - **Debuggable:** Modern async-await API allows breakpoints in asynchronous tests to work as expected.
 - **Automatically Synchronized:** Stops flakiness at the core by monitoring asynchronous operations in your app.
 - **Made For CI:** Execute your E2E tests on CI platforms like Travis CI, Circle CI or Jenkins without grief.
 - **Runs on Devices:** Gain confidence to ship by testing your app on a device/simulator just like a real user (not yet supported on iOS).
 - **Test Runner Agnostic:** Detox provides a set of APIs to use with any test runner without it. It comes with [Jest](https://jestjs.io) integration out of the box.
 
-## Supported Versions
+## Supported React Native Versions
 
-### Environment
+Detox was built from the ground up to support React Native projects.
+While Detox should work out of the box with almost any React Native version from the past few minor releases, official support is provided for React Native versions `0.70.x` and `0.71.x` without React Native's ["New Architecture"](https://reactnative.dev/docs/the-new-architecture/landing-page).
```

Comment:
@d4vidi regarding your comment - What do you think about:
> We officially support Detox for React Native versions **0.70.x** and **0.71.x**, without the ["New Architecture"](https://reactnative.dev/docs/the-new-architecture/landing-page). 

TBH I'm not a huge fan of the table since there is really no difference between iOS and Android in terms of our official support. If you want some border that will wrap the version numbers I can make a 1x1 table 😄